vector/src/main/java/im/vector/app/features/settings/VectorSettingsAdvancedSettingsFragment.kt

```diff
@@ -82,5 +87,17 @@ class VectorSettingsAdvancedSettingsFragment : VectorSettingsBaseFragment() {
         } else {
             findPreference<VectorPreferenceCategory>("SETTINGS_RAGE_SHAKE_CATEGORY_KEY")!!.isVisible = false
         }
+
+        bindMemoryLeakSetting()
+    }
+
+    private fun bindMemoryLeakSetting() {
+        findPreference<VectorSwitchPreference>(VectorPreferences.SETTINGS_ENABLE_MEMORY_LEAK_ANALYSIS_KEY)?.let { pref ->
+            pref.isVisible = BuildConfig.DEBUG
```

Comment:
to help the modularisation we've started avoiding direct usages of `BuildConfig`, `buildMeta.isDebug` is the replacement for `DEBUG`

Reply:
Okay thanks for the info. I will update it.